### DISCUSSION TITLE
자주 물어보는 질문 Q&A

### DISCUSSION THREAD

**Discussion by byungmeo** [byungmeo/GersangStation#3] [Announcements]
문의는 언제나 환영입니다. -> [문의 오픈톡방](https://open.kakao.com/o/sXJQ1qPd)

**Q. 다클라에 패치가 자동으로 적용이 안됩니다.**
A. #8

**Q. 프로그램 업데이트마다 계정을 다시 입력하기가 너무 귀찮아요.**
A. #13

**Q. 해킹에 대한 우려 때문에 거상 스테이션을 패치 용도로만 사용하고 싶어요.**
A. #14

**Q. 아이디나 비밀번호 입력 시 이상하게 입력됩니다. (키보드 자판 이상하게 입력됨)**
A. #9

**Q. 프로그램을 바탕화면에서 실행하려면 어떻게 해야하나요?**
A. #2 

**Q. SNS (네이버, 카카오, 구글) 유저는 거상 스테이션 사용 못하나요?**
A. #59 

**Q. 오류 발생 (0x80007139f)**
A. #55 

**Q. 경로 미지정**
A. 테섭 체크되어있는지 확인

**Comment by qogus10** [comment]
좋은 프로그램, 감사합니다 ^^
새해 복 많이 받으셔요.

열렙 득템 ~~
